src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/TabularLinearFunctions.hpp

```diff
@@ -0,0 +1,11 @@
+struct TabularLinearFunctions {
```

Comment:
You have to look at this one again. The ACEify function doesn't ACEify this correctly and there is no test to tell you so.

Reply:
Updated and working properly.